nextcord/abc.py

```diff
@@ -730,7 +730,13 @@ async def set_permissions(
     ) -> None:
         ...
 
-    async def set_permissions(self, target, *, overwrite=MISSING, reason=None, **permissions):
+    async def set_permissions(
+        self,
+        target: Union[Member, Role],
+        *,
+        reason: Optional[str] = None,
+        **kwargs: Any,
```

Comment:
They are both optional and keyword only args, so
```py
overwrite: Optional[PermissionOverwrite] = MISSING,
**permissions: bool,
```
should work just fine right?

Reply:
My apologies for not elaborating on why I used a catch-all variable keyword parameter.
You see, if I add what you're suggesting, I get this type error: 
```
Overloaded implementation is not consistent with signature of overload 2
  Type "(self: Self@GuildChannel, target: Member | Role, *, reason: str | None = None, overwrite: PermissionOverwrite | None = MISSING, **permissions: bool) -> Coroutine[Any, Any, None]" cannot be assigned to type "(self: Self@GuildChannel, target: Member | Role, *, reason: str | None = ..., **permissions: bool) -> Coroutine[Any, Any, None]"
    Parameter 5: type "bool" cannot be assigned to type "PermissionOverwrite | None"
      Type "bool" cannot be assigned to type "PermissionOverwrite | None"
        "bool" is incompatible with "PermissionOverwrite"
        Type cannot be assigned to type "None"
```